src/bp-templates/bp-nouveau/buddypress/common/js-templates/messages/parts/bp-messages-thread.php

```diff
@@ -25,23 +25,23 @@
                 if( data.avatars.length == 2 ) { #>
                     <div class="thread-multiple-avatar">
                 <# } #>
-                    <img class="avatar" src="{{data.avatars[0].url}}" alt="{{data.avatars[0].name}}"/>
+                    <img class="avatar 1" src="{{{data.avatars[0].url}}}" alt="{{data.avatars[0].name}}"/>
```

Comment:
```suggestion
                    <img class="avatar" src="{{{data.avatars[0].url}}}" alt="{{data.avatars[0].name}}"/>
```
I think `1` is for your debug purpose you added so need to remove.